Assets/Mirror/Editor/Weaver/Weaver.cs

```diff
@@ -627,9 +619,6 @@ static bool Weave(string assName, IEnumerable<string> dependencies, IAssemblyRes
                     WriterParameters writeParams = Helpers.GetWriterParameters(readParams);
                     CurrentAssembly.Write(dest, writeParams);
                 }
-
-                if (CurrentAssembly.MainModule.SymbolReader != null)
-                    CurrentAssembly.MainModule.SymbolReader.Dispose();
             }
```

Comment:
Notice everything including the SymboReader is disposed here because it falls out of the "using"  statement